pkg/metrics/metrics.go

```diff
@@ -258,8 +258,9 @@ var (
 	// LabelValuesBool is metric label value set for boolean type.
 	LabelValuesBool = metric.NewValues(LabelValueTrue, LabelValueFalse)
 
-	// Namespace is used to scope metrics from cilium. It is prepended to metric
-	// names and separated with a '_'
+	// Namespace is used to scope metrics from the current cilium component by
+	// overwriting its value at runtime. It is prepended to metric names and
```

Comment:
Nit. The caveat is that it can only be changed very early during initialization, before actually starting the hive, otherwise it would not have any effect.

Reply:
Thanks! I tried adding a little clarification that this is done before hive start